packages/web/src/components/toast/mobile/Toast.tsx

```diff
@@ -44,13 +48,27 @@ const Toast = (props: ToastProps) => {
   return (
     <>
       <div
-        className={cn(styles.wrapperClass, {
+        className={cn({
           [containerClassName!]: !!containerClassName
         })}
       >
         {children}
       </div>
-      <div className={styles.container}>{content}</div>
+      <Flex
+        direction='row'
+        gap='s'
+        alignItems='center'
+        backgroundColor='accent'
+        pv='s'
+        ph='m'
+        borderRadius='m'
+      >
+        {LeftIcon && <LeftIcon size='s' color='white' />}
```

Comment:
is `white` the color we want here? not `staticWhite` or something?

Reply:
probably? figma is unclear. ill do static white for now